experimental/iterators/lib/Conversion/IteratorsToLLVM/IteratorsToLLVM.cpp

```diff
@@ -1621,6 +1769,12 @@ static void convertIteratorOps(ModuleOp module, TypeConverter &typeConverter) {
                  "Expected iterator op to be converted to one value.");
           mapping.map(op->getResult(0), converted[0]);
         })
+        .Case<StreamToValueOp>([&](auto op) {
```

Comment:
Can we add some comments in this case and the previous one on the types that are expected before/after conversion?

Reply:
Solved in latest commit.